pkg/identity/issuer.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package identity
+
+import "context"
+
+type Issuer interface {
+	// Can this Issuer authenticate the given issuer URL?
+	Match(ctx context.Context, url string) bool
```

Comment:
Should this return an error or bool?

Reply:
Bool seems more reasonable to me as its not an error if the URL doesn't match

Reply:
SG, just wanted to check that it didn't also parse the URL or do anything that could return an error.

Reply:
Nope no parsing or anything necessary. Even in the meta providers its just a regex or string match kinda thing so no errors there either. It just matches or doesn't match